lessons/introR/main.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Introduction to R"
+author: "Luke Johnston"
+date: "July 23, 2015"
+output: 
+  html_document: 
+    highlight: tango
+    number_sections: yes
+    theme: readable
+    toc: yes
+    
+---
+
+This is a brief introduction to R, focussing on data wrangling using
+`dplyr` and `tidyr` packages and generating reproducible documents
+using `knitr` and `rmarkdown` packages.
+
+You also **don't need to read all of this** for the session.  It's
+more of a **resource/reference**.
+
+R is a statistical computing environment to analyze data and write
+programs.  The strength of R comes from:
+
+* It being developed by statisticians to do statistical analysis,
+  which is the primary reason to use R.
+* It's graphic capacities are top-quality
+* It has excellent resources for creating reproducible documents.
+* It's extensive and active community of users for doing statistical
+  work:
+    - R is the top tag in the
+      [StackExchange CrossValidated site](stats.stackexchange.com/tags)
+      for statistical questions
+    - Has more than 6000 packages on
+      [CRAN](https://cran.r-project.org/web/packages/) that stores all
+      R packages (and installing these packages is straight-forward),
+      from the obscure and cutting-edge statistical techniques to
+      plotting to data wrangling.
+
+I should mention a quick caveat.  R is a programming
+language... however it works very differently from other
+general-purpose programming languages ie. Python (it was developed by
+statisticians after all!).  As such, programming in R can be not as
+intuitive, powerful, or easy as it may be in Python (though it can be
+done).  If your work involves a lot of programming, I would recommend
```

Comment:
I would qualify that as "general-purpose programming", unless you don't consider R a programming language at all (but it probably is, just a domain-specific one.)